shorten

```diff
@@ -20,6 +20,10 @@ done
 
 case "${book}" in
   chemistry) dest="${DIR}/data/chemistry/short" && script="${DIR}/books/chemistry/shorten";;
+  sociology) dest="${DIR}/data/sociology/short" && script="${DIR}/books/sociology/shorten";;
+  # sociology) dest="${DIR}/data/sociology-2e/short" && script="${DIR}/books/sociology/shorten";;
+  # sociology) dest="${DIR}/data/sociology-3e/short" && script="${DIR}/books/sociology/shorten";;
```

Comment:
I have used it as a temporarily solution for using shorten script on two sociology editions.